torch/_torch_docs.py

```diff
@@ -1500,6 +1500,58 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov,
+           r"""
+cov(input1, input2=None, rowvar=True, bias=None, ddof=None, fweights=None, aweights=None) -> Tensor
+
+Estimates a covariance matrix, given input tensor data and weights (optional).
+
+Args:
+    input1 (Tensor): A 1-D or 2-D tensor containing multiple variables and observations. 
+        Each row of :attr:`input1` represents a variable, and each column a 
+        single observation of all those variables. Also see :attr:`rowvar` below.
+    input2 (Tensor, optional): An additional set of variables and observations. 
+        :attr:`input2` has the same form as that of :attr:`input1`.
+    rowvar (bool, optional): If :attr:`rowvar` is `True` (default), then each row represents a variable, 
+        with observations in the columns. 
+        Otherwise, the relationship is transposed: each column represents a variable, 
+        while the rows contain observations.
+    bias (bool, optional): Default normalization (`False`) is by :math:`(N - 1)`, 
+        where :math:`N` is the number of observations given (unbiased estimate). 
```

Comment:
The parenthetical "unbiased estimate" is good but this should elaborate to clarify precisely what's meant (an unbiased estimate of what?). A mathematical expression at the start of the function might help clarify the relation between bias and the computation.